### PR TITLE
support Image for sample code

### PR DESCRIPTION
**Pull request by zhouhongmei89**
## Purpose
support image function
* ...

## Pull Request Type
add function

### CODE REVIEW

**Comment by zhouhongmei89** [comment]
@microsoft-github-policy-service agree company="Microsoft"


Best Regards~
mail: ***@***.***)
Phone Number:18862179331

File: samples/js/browser/avatar/js/chat.js [hunk 1]
```diff
@@ -348,10 +349,25 @@ function stopSpeaking() {
     )
 }
 
-function handleUserQuery(userQuery) {
+function handleUserQuery(userQuery,userQueryHTML,imgUrlPath) {
```
Comment (yinhew) [inline samples/js/browser/avatar/js/chat.js:352]: For better readability, please put a white space after the ","

File: samples/js/browser/avatar/js/chat.js [hunk 2]
```diff
@@ -689,17 +707,40 @@ window.updataEnableOyd = () => {
...
+                    childImg1.style.height = "200px"
+                }
+                let userQueryHTML = messageBox.innerHTML.trim("")
+                if(userQueryHTML.startsWith('<img')){
+                    userQueryHTML="<br/>"+userQueryHTML
+                }
                 if (userQuery !== '') {
-                    handleUserQuery(userQuery.trim('\n'))
-                    document.getElementById('userMessageBox').value = ''
+                    handleUserQuery(userQuery.trim(''), userQueryHTML,imgUrl)
```
Comment (yinhew) [inline samples/js/browser/avatar/js/chat.js:725]: For better readability, please put a white space after the ","

File: samples/js/browser/avatar/js/chat.js [hunk 3]
```diff
@@ -689,17 +707,40 @@ window.updataEnableOyd = () => {
 window.updateTypeMessageBox = () => {
     if (document.getElementById('showTypeMessage').checked) {
         document.getElementById('userMessageBox').hidden = false
+        document.getElementById('uploadImgIcon').hidden = false
         document.getElementById('userMessageBox').addEventListener('keyup', (e) => {
             if (e.key === 'Enter') {
-                const userQuery = document.getElementById('userMessageBox').value
+                const userQuery = document.getElementById('userMessageBox').innerText
+                const messageBox = document.getElementById('userMessageBox')
+                const childImg1 = messageBox.querySelector("#picInput")
```
Comment (yinhew) [inline samples/js/browser/avatar/js/chat.js:715]: Can you rename childImg1 to childImg ?

**Pull request merged**